lib/utils.pm

```diff
@@ -1892,6 +1893,15 @@ sub install_patterns {
             $pt .= '*';
             $pcm = 1;
         }
+        # Only one CFEngine pattern can be installed
+        if ($pt =~ /CFEngine|CFEngien/) {
```

Comment:
Lemon, what is "CFEngien"? a typo?

Reply:
There are two similar pattern CFEngine and CFEngien, only one should be selected for all patterns test.

Reply:
Why use 'CFEngien'. Don't understand.

Reply:
I don't know why define as CFEngien, but it is just a pattern's name, no special.